internal/app/commands/common/common.go

```diff
@@ -212,14 +217,27 @@ func EnsureAuth(authString string, force bool) (string, string) {
 	return splitAuth[0], splitAuth[1]
 }
 
+func resolveCredFile(path string) string {
+	pathFromEnv := os.Getenv("ENONIC_CLI_CRED_FILE")
+	if pathFromEnv != "" {
+		return pathFromEnv
+	} else {
+		return path
+	}
```

Comment:
In other parts of the app we give priority to command line arguments over the env vars, so this probably needs to be swapped too

Reply:
done